docs/howto/README.md

```diff
@@ -4,6 +4,12 @@ Here you’ll find short answers to "How do I...?" types of questions. These how
 topics in depth – you’ll find that material in the [topic guides](../topics/README.md) section.
 However, these guides will help you quickly accomplish common tasks.
 
+Generic approach for any Kubernetes cluster (using inlets):
+
+- [Running Krustlet on Kubernetes with inlets](krustlet-on-kubernetes-with-inlets.md)
+
+Specific instructions for `krustlet` on various Kubernetes distributions:
```

Comment:
Can we remove this? It's intended to read like a table of contents without any preface material. Right now the HOWTO guides only demonstrate how to get started with Krustlet/Kubernetes, but there may be more WebAssembly-specific HOWTOs in the future.

Reply:
We can remove it however it won't be clear as to why multiple guides exist.